models/articles.models.js

```diff
@@ -49,17 +49,26 @@ exports.selectArticles = (topic) => {
 };
 
 exports.selectArticleById = (id) => {
-  return db
-    .query(`SELECT * FROM articles WHERE article_id = $1`, [id])
-    .then(({ rows }) => {
-      if (!rows[0]) {
+  const article = db.query(`SELECT * FROM articles WHERE article_id = $1`, [
+    id,
+  ]);
+  const commentCount = db.query(
+    `SELECT COUNT(comment_id) FROM comments WHERE article_id = $1`,
+    [id]
+  );
```

Comment:
Getting the comment_count should be achieved using a single sql query. Trying refactoring this as it would be good sql practice 🙂